### PR TITLE
WIP: rename fields in PSP

### PR DESCRIPTION
**Pull request by thockin**
not for merge

### CODE REVIEW

File: pkg/apis/extensions/v1beta1/types.go [hunk 1]
```diff
@@ -968,27 +968,27 @@ type HostPortRange struct {
...
-type SELinuxContextStrategyOptions struct {
+// SELinux Strategy Options defines the strategy type and any options used to create the strategy.
+type SELinuxStrategyOptions struct {
 	// type is the strategy that will dictate the allowable labels that may be set.
-	Type SELinuxContextStrategy `json:"type"`
-	// seLinuxOptions required to run as; required for MustRunAs
+	Type SELinuxStrategy `json:"type"`
+	// Options required to run as; required for MustRunAs
 	// More info: http://releases.k8s.io/HEAD/docs/design/security_context.md#security-context
-	SELinuxOptions *v1.SELinuxOptions `json:"seLinuxOptions,omitempty"`
```
Comment (erictune) [inline pkg/apis/extensions/v1beta1/types.go]: this is called `seLinuxOptions` in the `container[].securityContext`.  Do you like shortness in one place or consistency in two places?

Comment (thockin) [inline pkg/apis/extensions/v1beta1/types.go]: I'm trying to follow the go "don't stutter" rule.  If the type already says
"selinux" and the JSON path to the field says "selinux", it's nice if we
can not have to say "selinux" again.  Not a huge deal, certainly not
something I am going to fight over.

Comment (erictune) [inline pkg/apis/extensions/v1beta1/types.go]: I knew that was why you suggested the change.  I just didn't know whether you were aware of the inconsistency, and whether you care about consistency or stuttering more.

Comment (thockin) [inline pkg/apis/extensions/v1beta1/types.go]: In this case it's a new API object and the "other" feels far enough away that doing the right thing here seems more useful than consistency.  It does break cut-paste, so I can see it both ways.  I would go with shorter myself, but I am probably not the person who will be answering emails about why it doesn't work.

**Comment by thockin** [comment]
I'll close this in favor of yours

**Pull request closed without merging**